vcd/datasource_vcd_nsxt_edge_cluster.go

```diff
@@ -18,9 +22,29 @@ func datasourceVcdNsxtEdgeCluster() *schema.Resource {
 					"level. Useful when connected as sysadmin working across different organizations",
 			},
 			"vdc": {
-				Type:        schema.TypeString,
-				Optional:    true,
-				Description: "The name of VDC to use, optional if defined at provider level",
+				Type:          schema.TypeString,
+				Optional:      true,
+				Description:   "The name of VDC to use, optional if defined at provider level",
+				ConflictsWith: []string{"vdc_id", "vdc_group_id", "provider_vdc_id"},
+				Deprecated:    "This field is deprecated in favor of 'owner_id' which accepts IDs of VDC, VDC Group and Provider VDC",
+			},
+			"vdc_id": {
+				Type:          schema.TypeString,
+				Optional:      true,
+				Description:   "ID of VDC, VDC Group or Provider VDC",
+				ConflictsWith: []string{"vdc", "vdc_group_id", "provider_vdc_id"},
+			},
+			"vdc_group_id": {
+				Type:          schema.TypeString,
+				Optional:      true,
+				Description:   "ID of VDC, VDC Group or Provider VDC",
```

Comment:
Should it be
```suggestion
				Description:   "ID of VDC Group",
```
?

Reply:
Yes! I messed this up from initial mockups of code.